packages/components/releaseNotes/components.md

```diff
@@ -1,6 +1,10 @@
 # @labkey/components
 Components, models, actions, and utility functions for LabKey applications and pages.
 
+### version 2.137.4
```

Comment:
FYI, I usually will put TBD as the version number and release date while doing dev and then fill that in right before doing the final publish / merge.